<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>piano</class>
 <widget class="QMainWindow" name="piano">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>997</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>piano</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="C">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="D">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Db">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>60</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Db</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Eb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>20</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Eb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="E">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Gb">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="G">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ab">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Ab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bb">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Bb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>170</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start recording</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>170</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop recording</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>170</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create csv file</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>401</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="C_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>480</y>
      <width>170</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Compose from csv</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Make your own music !&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>720</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Step 1. Play around with the given virtual piano below!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>750</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 2. Record the set of notes that you want the AI to generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>790</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>music from.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>820</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 3. Create csv file, compose it and follow the given 
            instructions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="D_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="E_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Gb_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="C_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Eb_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>20</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Eb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ab_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Ab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bb_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Bb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Db_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>20</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Db</string>
    </property>
   </widget>
   <widget class="QPushButton" name="G_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="D_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="E_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Gb_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Gb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="C_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Eb_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>20</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Eb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="F_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ab_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Ab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bb_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>21</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Bb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Db_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>20</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Db</string>
    </property>
   </widget>
   <widget class="QPushButton" name="G_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>670</y>
      <width>170</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear recorded csv</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>580</y>
      <width>170</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save music !</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>660</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Step 0. Make sure you made csvmidi-midicsv by running make in 
            the command prompt in csvmidi-midicsv folder </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>860</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 4. Save your music and have a listen!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
